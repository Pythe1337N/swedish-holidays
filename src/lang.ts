export const swedishHolidayNames = {
  newYearsDay: "Nyårsdagen",
  twelfthNight: "Trettondagsafton",
  epiphany: "Trettondedag jul",
  maundyThursday: "Skärtorsdagen",
  goodFriday: "Långfredagen",
  holySaturday: "Påskafton",
  easterSunday: "Påskdagen",
  easterMonday: "Annandag påsk",
  walpurgisNight: "Valborgsmässoafton",
  mayFirst: "Första maj",
  ascensionDay: "Kristi himmelsfärdsdag",
  swedishNationalDay: "Sveriges nationaldag",
  pentecostEve: "Pingstafton",
  whitSunday: "Pingstdagen",
  midsummerEve: "Midsommarafton",
  midsummerDay: "Midsommardagen",
  allSaintsEve: "Allhelgonaafton",
  allSaintsDay: "Alla helgons dag",
  christmasEve: "Julafton",
  christmasDay: "Juldagen",
  boxingDay: "Annandag jul",
  newYearsEve: "Nyårsafton",
};

export const sunday = "Sōndag";

export type LanguageType = typeof swedishHolidayNames;
