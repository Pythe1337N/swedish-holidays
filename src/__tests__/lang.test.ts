import { test, expect } from "vitest";
import { swedishHolidayNames } from "../lang";

test(`Test swedishHolidayNames`, () => {
  expect(swedishHolidayNames).toMatchSnapshot();
});
