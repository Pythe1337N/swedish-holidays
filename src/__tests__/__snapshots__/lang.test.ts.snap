// Vitest Snapshot v1

exports[`Test swedishHolidayNames 1`] = `
{
  "allSaintsDay": "Alla helgons dag",
  "allSaintsEve": "Allhelgonaafton",
  "ascensionDay": "Kristi himmelsfärdsdag",
  "boxingDay": "Annandag jul",
  "christmasDay": "Juldagen",
  "christmasEve": "Julafton",
  "easterMonday": "Annandag påsk",
  "easterSunday": "Påskdagen",
  "epiphany": "Trettondedag jul",
  "goodFriday": "Långfredagen",
  "holySaturday": "Påskafton",
  "maundyThursday": "Skärtorsdagen",
  "mayFirst": "Första maj",
  "midsummerDay": "Midsommardagen",
  "midsummerEve": "Midsommarafton",
  "newYearsDay": "Nyårsdagen",
  "newYearsEve": "Nyårsafton",
  "pentecostEve": "Pingstafton",
  "swedishNationalDay": "Sveriges nationaldag",
  "twelfthNight": "Trettondagsafton",
  "walpurgisNight": "Valborgsmässoafton",
  "whitSunday": "Pingstdagen",
}
`;
